seacrowd/sea_datasets/etos/etos.py

```diff
@@ -0,0 +1,206 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import conllu
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@INPROCEEDINGS{10053062,
+  author={Samsuri, Mukhlizar Nirwan and Yuliawati, Arlisa and Alfina, Ika},
+  booktitle={2022 5th International Seminar on Research of Information Technology and Intelligent Systems (ISRITI)},
+  title={A Comparison of Distributed, PAM, and Trie Data Structure Dictionaries in Automatic Spelling Correction for Indonesian Formal Text},
+  year={2022},
+  pages={525-530},
+  keywords={Seminars;Dictionaries;Data structures;Intelligent systems;Information technology;automatic spelling correction;distributed dictionary;non-word error;trie data structure;Partition Around Medoids},
+  doi={10.1109/ISRITI56927.2022.10053062},
+  url = {https://ieeexplore.ieee.org/document/10053062},
+}
+"""
+
+_DATASETNAME = "etos"
+
+_DESCRIPTION = """\
+ETOS (Ejaan oTOmatiS) is a dataset for parts-of-speech (POS) tagging for formal Indonesian
+text. It consists of 200 sentences, with 4,323 tokens in total, annotated following the
+CoNLL format.
+"""
+
+_HOMEPAGE = "https://github.com/ir-nlp-csui/etos"
+
+_LANGUAGES = {"ind": "id"}
```

Comment:
```suggestion
_LANGUAGES = ["ind"]
```

Our template uses list of ISO639-3 codes.